### PR TITLE
Misc

### PR DESCRIPTION
**Pull request by cgzones**

### CODE REVIEW

File: ProcessList.c [hunk 1]
```diff
@@ -564,7 +564,7 @@ void ProcessList_rebuildPanel(ProcessList* this) {
 
 Process* ProcessList_getProcess(ProcessList* this, pid_t pid, bool* preExisting, Process_New constructor) {
    Process* proc = (Process*) Hashtable_get(this->processTable, pid);
-   *preExisting = proc;
+   *preExisting = (proc != NULL);
```
Comment (BenBE) [inline ProcessList.c:567]: Unnecessary parens …

File: Process.c [hunk 2]
```diff
@@ -617,6 +617,7 @@ int Process_compareByKey_Base(const Process* p1, const Process* p2, ProcessField
    case USER:
       return SPACESHIP_NULLSTR(p1->user, p2->user);
    default:
+      assert(0); /* should never be reached */
```
Comment (BenBE) [inline Process.c:620]: Maybe use comma operator to provide some meaningful message in case of assertion failure.

File: ProcessList.c (hunk 1 above)
Comment (cgzones) [reply ProcessList.c]: removed

File: Process.c (hunk 2 above)
Comment (cgzones) [reply Process.c]: added

**Pull request merged**